lib/peddler-ratelimited/create_inbound_shipment_plan.rb

```diff
@@ -0,0 +1,71 @@
+module PeddlerRateLimited
+  class CreateInboundShipmentPlan < AmazonMWSApi
+
+    SUBJECT = 'create_inbound_shipment_plan'
+    BURST_RATE = 30
+    RESTORE_RATE = 0.5 #half a second
+    MAX_HOURLY_RATE = 7200
+
+    @backoff_strategy = (1..5).map {|i| i*RESTORE_RATE }
+
+    @queue = :amazon_api_create_inbound_shipment_plan
+
+    def self.perform(args = {})
+      args.deep_symbolize_keys!
+      RateLimitter.new(self, args).submit
+    end
+
+    def self.act(args)
+      binding.pry
+      result = call_feed(args)
+      binding.pry
+
+      process_feeds_list(result, args[:processor])
+    rescue Exception => e
+      binding.pry
+      log_error('create_inbound_shipment_plan', result, e, args)
+    end
+
+    def self.call_feed(args)
+      AmazonMWS.instance.
+        inbound_fulfillment.
+        create_inbound_shipment_plan(args[:ship_from_address],
```

Comment:
are these args object?

Reply:
yes, array of hashes but it looks like Resque is ok with them.